custom/icds_reports/reports/issnip_monthly_register.py

```diff
@@ -371,7 +359,123 @@ def agg_child_health_monthly(self):
+            }, 'rations_21_plus_distributed')),
+            pre_sc_girls_36_72=Sum(self.filter_by({
+                'caste': 'sc',
+                'gender': 'F',
+                'age_tranche__in': ['48', '60', '72']
+            }, 'rations_21_plus_distributed')),
+            pre_st_boys_36_72=Sum(self.filter_by({
+                'caste': 'st',
+                'gender': 'M',
+                'age_tranche__in': ['48', '60', '72']
+            }, 'rations_21_plus_distributed')),
+            pre_st_girls_36_72=Sum(self.filter_by({
+                'caste': 'st',
+                'gender': 'F',
+                'age_tranche__in': ['48', '60', '72']
+            }, 'rations_21_plus_distributed')),
+            pre_obc_boys_36_72=Sum(self.filter_by({
+                'caste': 'obc',
+                'gender': 'M',
+                'age_tranche__in': ['48', '60', '72']
+            }, 'rations_21_plus_distributed')),
+            pre_obc_girls_36_72=Sum(self.filter_by({
+                'caste': 'obc',
+                'gender': 'F',
+                'age_tranche__in': ['48', '60', '72']
+            }, 'rations_21_plus_distributed')),
+            pre_general_boys_36_72=Sum(self.filter_by({
+                'caste': 'other',
+                'gender': 'M',
+                'age_tranche__in': ['48', '60', '72']
+            }, 'rations_21_plus_distributed')),
+            pre_general_girls_36_72=Sum(self.filter_by({
+                'caste': 'other',
+                'gender': 'F',
+                'age_tranche__in': ['48', '60', '72']
+            }, 'rations_21_plus_distributed')),
+            pre_total_boys_36_72=Sum(self.filter_by({
+                'gender': 'M',
+                'age_tranche__in': ['48', '60', '72']
+            }, 'rations_21_plus_distributed')),
+            pre_total_girls_36_72=Sum(self.filter_by({
+                'gender': 'F',
+                'age_tranche__in': ['48', '60', '72']
+            }, 'rations_21_plus_distributed')),
+            pre_minority_boys_36_72=Sum(self.filter_by({
+                'gender': 'M',
+                'age_tranche__in': ['48', '60', '72'],
+                'minority': 'yes'
+            }, 'rations_21_plus_distributed')),
+            pre_minority_girls_36_72=Sum(self.filter_by({
+                'gender': 'F',
+                'age_tranche__in': ['48', '60', '72'],
+                'minority': 'yes'
+            }, 'rations_21_plus_distributed')),
+
+        )
+        return {row['awc_id']: row for row in data}
 
+    @cached_property
+    def agg_ccs_record_monthly(self):
+        data = AggCcsRecordMonthly.objects.filter(
+            awc_id__in=self.config['awc_id'],
+            aggregation_level=AWC_LOCATION_LEVEL,
+            month=self.config['month']
+        ).values('awc_id').annotate(
+            sc_pregnant=Sum(self.filter_by({
+                'caste': 'sc',
+                'pregnant': 1
+            }, 'rations_21_plus_distributed')),
```

Comment:
These filters should look like `ccs_status = pregnant/lactating` instead of `pregnant/lactating = 1`